doc/howtos/realsense.rst

```diff
@@ -0,0 +1,34 @@
+Intel® RealSense™ cameras in IoT Reference OS Kit
```

Comment:
Well, ReST supports substitutions via:

```
.. |copy|   unicode:: U+000A9 .. COPYRIGHT SIGN
   :ltrim:
.. |trade|  unicode:: U+02122 .. TRADEMARK SIGN
   :ltrim:
.. |reg|    unicode:: U+000AE .. REGISTERED TRADEMARK SIGN
   :ltrim:
```
and GitHub will render these properly if entered as ``Intel |reg|``.  But GitHub doesn't support the ``.. include`` directive so you'd have to add the substitutions directives within each file that wanted to use them.

That said, using the unicode symbols such as ``®`` and such works just fine for GitHub display and if we eventually use a Sphinx-generation scheme.